dbms/src/Storages/Page/V3/PageDirectoryFactory.cpp

```diff
@@ -135,6 +135,11 @@ bool PageDirectoryFactory::applyRecord(
         iter->second = std::make_shared<VersionedPageEntries>();
     }
 
+    if (r.page_id.low > dir->max_page_id)
```

Comment:
Can we use `max_applied_page_id` with some modifications of https://github.com/pingcap/tiflash/blob/2e8d324e76a2c5a96c83fdab8e7e08f3a5424d61/dbms/src/Storages/Page/V3/PageDirectoryFactory.cpp#L114

Reply:
removed